.github/workflows/triage.yml

```diff
@@ -6,11 +6,10 @@ on:
       - opened
 
 env:
-  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
```

Comment:
Commenting on it to remember that we need to change this *before* we merge.

Reply:
Looks like we already have it there. Last updated in September though - hopefully it's up to date.